mandatory/2-function-creation.js

```diff
@@ -45,7 +83,12 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  while (number >= 1) {
```

Comment:
Suggestion: create a new variable that you can loop over and decrement rather than using the input `number`. e.g.
```
let currentNumber = number;
while (currentNumber  >= minNum) {
  console.log(number);
  currentNumber--;
}
```
The reason for this is you still have access to the unmodified user input in case you need to use it in  a future modification of the function.
Also you'll find in the future lessons that for certain types passed to a function modifying the input data could have an impact on other parts of the program (this isn't a problem for integers though so wouldn't be an issue here)

Reply:
Thanks, Michael, I think I need you to explain this a bit more to me over a buddy chat so I'll bring this up.